Plugins/Public/base_plugin/PlayerBase.cpp

```diff
@@ -314,7 +319,11 @@ void PlayerBase::Load()
 						else {
 							bp.admin = true;
 						}
-						passwords.push_back(bp);
+						passwords.emplace_back(bp);
+					}
+					else if (ini.is_value("crew_supplied"))
```

Comment:
what if this never gets called e.g. because we're reading a version of the base file written by an older version of the plugin? won't that cause problems with the default of isCrewSupplied being false?

Reply:
Since we have no way of knowing about 'Crew Status' prior to deployment of this patch, it'll have to be false until the first FOW check occurs after booting up the server post-patch.

It's a minor one-time inconvenience that we should be more than fine with as there are no real workarounds that I can see.